docker-compose.yml

```diff
@@ -48,8 +48,24 @@ services:
       - neo4j-logs:/logs
       - neo4j-import:/var/lib/neo4j/import
       - neo4j-plugins:/plugins
+  postgres:
+    container_name: mcs-postgres
+    environment:
+      POSTGRES_USER: "mcs"
+      POSTGRES_PASSWORD: "7EAdu7jJvZNxxrNZ"
+      POSTGRES_DB: "kg"
+    image: library/postgres:12.4
+    networks:
+      - mcs
+    ports:
+      - 5432:5432
+    restart: unless-stopped
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+
 volumes:
   neo4j-data:
```

Comment:
You'll want to merge master. I've modified this docker-compose.yml a few times to support the ISI installation.

Reply:
Gotcha, I will hold off merging for now to avoid cluttering up the reviews